content/elections-2023.md

```diff
@@ -140,33 +128,21 @@ donate ="""
 >}}
 """
 
-[[internal-links]]
-title = "Voter Guides"
-link = "guides"
-
-[[internal-links]]
-title = "Issues"
-link = "court-tool"
-
-[[internal-links]]
-title = "Election Event Series"
-link = "events"
-
-[[internal-links]]
-title = "Coverage"
-link = "coverage"
-
-# [[internal-links]]
-# title = "Español"
-# link = "#espan"
-
-[[internal-links]]
-title = "Ask Us"
-link = "contact"
+[[blocks]]
+title = "PA Election 2023: Candidate and Voting Guides"
+slug = "guides"
+layout = "story-collection"
+timeFilter = "2023-08-01"
+collection = "content/series/voting-guides-2023/_index.md"
```

Comment:
It seems like this isn't working. I don't know if it's related, but with Hugo 123, you need to omit the "content/" at the front.